keras_tuner/engine/oracle.py

```diff
@@ -394,19 +398,20 @@ def get_best_trials(self, num_trials=1):
         return sorted_trials[:num_trials]
 
     def remaining_trials(self):
-        if self.max_trials:
-            return self.max_trials - len(self.trials.items())
-        else:
-            return None
+        return (
+            self.max_trials - len(self.trials.items()) if self.max_trials else None
+        )
 
     def get_state(self):
```

Comment:
Sorry, I don't get this comment.

Reply:
This function should look like:
```py
    def get_state(self):
        return {
            "ongoing_trials": {
                tuner_id: trial.trial_id
                for tuner_id, trial in self.ongoing_trials.items()
            },
            # Hyperparameters are part of the state because they can be added to
            # during the course of the search.
            "hyperparameters": self.hyperparameters.get_config(),
            "seed": self.seed,
            "seed_state": self._seed_state,
            "tried_so_far": list(self._tried_so_far),
        }
```